### DISCUSSION TITLE
GitHub Copilot propose spaces as solution

### DISCUSSION THREAD

**Discussion by AuracleTech** [community/community#7821] [Copilot Conversations]
### Issue
Sometimes GitHub Copilot propose spaces as solution, but it doesn't show spaces in italic ( obviously ) so it's purely confusing.
A space, that's not really there, so you delete it, and it deletes the next character.
![image](https://user-images.githubusercontent.com/40531871/142157164-65915ffd-b829-4b2f-b1da-4f1d8be4a5b2.png)
Try to guess if the space next to my text cursor is there or just a prediction!

### Solution
Add a dot for spaces like when highlighting spaces ( see example ).
![image](https://user-images.githubusercontent.com/40531871/142156982-42bc9869-e63c-41b7-913c-493e21624dc3.png)

**Comment by MulleDK19** [comment]
You can hit escape to decline a suggestion to make sure it's not just a suggestion.

But yeah, that's a little weird.
